packages/tupaia-web/src/features/Matrix.tsx

```diff
@@ -125,19 +148,22 @@ interface MatrixProps {
   report: MatrixReport;
   isEnlarged?: boolean;
 }
+
 export const Matrix = ({ config, report, isEnlarged = false }: MatrixProps) => {
   const { columns = [], rows = [] } = report;
   const [searchFilter, setSearchFilter] = useState('');
 
   const placeholderImage = getPlaceholderImage(config);
-  // in the dashboard, show a placeholder image
-  if (!isEnlarged) return <img src={placeholderImage} alt="Matrix Placeholder" />;
 
-  const parsedRows = parseRows(rows, undefined, searchFilter);
+  const { periodGranularity, drillDown } = config;
+
+  const baseDrillDownLink = getBaseDrilldownLink(drillDown);
+  const parsedRows = parseRows(rows, undefined, searchFilter, drillDown, baseDrillDownLink);
```

Comment:
sheesh there is a lot going on in this component now but I think that it all fits in pretty nicely.